doc/user/content/sql/create-source/webhook.md

```diff
@@ -247,7 +291,7 @@ The first step for setting up a webhook source is to create a shared secret. Whi
 required, it's the recommended best practice.
 
 ```
-CREATE SECRET segment_shared_secret AS 'abc123';
+CREATE SECRET segment_shared_secret AS 'abc123'
```

Comment:
Looking at examples of other docs pages, it looks like we do typically end all example statements with a semicolon, so we should be consistent here too

Reply:
Good call out, I updated all of the examples to include semicolons!